docs/examples/cloudbuildsource/README.md

```diff
@@ -0,0 +1,235 @@
+# CloudBuildSource Example
+
+## Overview
+
+This sample shows how to configure `CloudBuildSources`. The `CloudBuildSource`
+fires a new event each time when your build's state changes, 
+such as when your build is created, 
+when your build transitions to a working state, 
+and when your build completes.
+
+## Prerequisites
+
+1. [Install Knative-GCP](../../install/install-knative-gcp.md)
+
+1. [Create a Pub/Sub enabled Service Account](../../install/pubsub-service-account.md)
+
+1. Enable the `Cloud Build API` and `Cloud Pub/Sub API`,  on your project:
+
+   ```shell
+   gcloud services enable cloudbuild.googleapis.com
+   gcloud services enable pubsub.googleapis.com
+   ```
+   The Pub/Sub topic to which Cloud Build publishes these build update messages is called `cloud-builds`. 
+   When you enable the `Cloud Build API` and `Cloud Pub/Sub API`, the `cloud-builds` topic will automatically be created for you. 
+
+## Deployment
+
+1. Create a [`CloudBuildSource`](cloudbuildsource.yaml)
+
+   1. If you are in GKE and using
+      [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity),
+      update `googleServiceAccount` with the Pub/Sub enabled service account you
+      created in
+      [Create a Pub/Sub enabled Service Account](../../install/pubsub-service-account.md).
+
+   1. If you are using standard Kubernetes secrets, but want to use a
+      non-default one, update `secret` with your own secret.
```

Comment:
What permissions does this secret need? Just PubSub subscriber?

Reply:
Yep. roles/pubsub.subscriber is enough.

Reply:
Please document that here, or link to the `Create a Pub/Sub enabled Service Account` page which says so.

Reply:
This is documented in the table at https://github.com/google/knative-gcp/pull/770/files#diff-601ad8f8536a2b3c34d484c5699a052dR120, but should be mentioned here too.